src/dash-table/reconcile/text.ts

```diff
@@ -0,0 +1,34 @@
+import { ITextColumn } from 'dash-table/components/Table/props';
+import isNully from './isNully';
+
+export function coerce(value: any, options?: ITextColumn) {
+    const validation = validate(value, options);
+    if (validation.success) {
+        return validation;
+    }
+
+    const allowNull = Boolean(options && options.validation && options.validation.allow_null);
+    const nully = isNully(value);
+
+    return {
+        success: Boolean(!nully || allowNull),
+        value: nully ? value : JSON.stringify(value)
```

Comment:
I was going to say that the nully case was already handled by `validate`, but I guess you're not stringifying nully values? Is that what we want for text/coerce? I would have thought that even they should coerce to strings.

Reply:
> If we need/want to support the conversion of nully values, it should be done through another prop.

That all makes sense, sold. Thanks for expanding on your thought process.